index.d.ts

```diff
@@ -0,0 +1,5 @@
+import {Plugin} from 'unified'
+
+declare const remarkSlug: Plugin
```

Comment:
the default generic on plugin accepts a single object like setting.
This plugin does not accept any settings, which can be specified with `<[]>`

```suggestion
declare const remarkSlug: Plugin<[]>
```